pjsip/include/pjsip-ua/sip_inv.h

```diff
@@ -388,7 +388,21 @@ enum pjsip_inv_option
      * Require trickle ICE support.
      */
     PJSIP_INV_REQUIRE_TRICKLE_ICE = 512,
+    
+    /**
+     * Require siprec support.
+     */
+    PJSIP_INV_REQUIRE_SIPREC      = 1024,
 
+    /**
+     * Indicate support for siprec
+     */
+    PJSIP_INV_SUPPORT_SIPREC      = 2048,
+    
+    /**
+     * Indicate not support for siprec
+     */
+    PJSIP_INV_NOT_SUPPORT_SIPREC  = 4096,
```

Comment:
`PJSIP_INV_NOT_SUPPORT_SIPREC` is required because when the user sets `use_siprec` to `PJSUA_SIP_SIPREC_INACTIVE`, I reject SIPREC calls with a Bad Extension error.

Reply:
By default, no flag means unsupported. Check the other flags above.